readthedocs/settings/docker_compose.py

```diff
@@ -225,6 +225,10 @@ def SOCIALACCOUNT_PROVIDERS(self):
                 pass
         return providers
 
+    GITHUB_APP_ID = os.environ.get("RTD_GITHUB_APP_ID")
+    GITHUB_APP_WEBHOOK_SECRET = os.environ.get("RTD_GITHUB_APP_WEBHOOK_SECRET")
+    GITHUB_APP_PRIVATE_KEY = os.environ.get("RTD_GITHUB_APP_PRIVATE_KEY")
```

Comment:
You will need to add these to the docker compose YAML file in `common/`, right?

Reply:
yep, I already have the changes locally